tools/pylint-extensions/pylint-guidelines-checker/pylint_guidelines_checker.py

```diff
@@ -1914,7 +1914,46 @@ def visit_module(self, node):
     
         except Exception:
                 logger.debug("Pylint custom checker failed to check if model is aliased.")
-                pass
+
+class BlockedImport(BaseChecker):
+    """There are certain imports that should only occur in the core package.
+    For example, instead of using `requests` to make requests, clients should
+    take a `azure.core.pipeline.Pipeline` as input to make requests.
+    """
+    name = "blocked-import"
+    priority = -1  # what is this?
```

Comment:
priority is " This must be to be lower than 0. The checkers are ordered by
the priority when run, from the most negative to the most positive." - pylint